### PR TITLE
Feature/pro 1035

### PR DESCRIPTION
**Pull request by ssoegriemsingh**

### CODE REVIEW

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class App {
+
+    String SPACE = " ";
+    Random RANDOM = new Random();
+    Scanner SCANNER = new Scanner(System.in);
```
Comment (ibereziuk) [inline 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java]: use upper case names for constant(final) objects only

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/ispokemon/isPokemon.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package ispokemon;
+
+public interface isPokemon {
```
Comment (ibereziuk) [inline 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/ispokemon/isPokemon.java]: Interface name should start from Capital

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    List<isPokemon> pokemonList = new ArrayList<>();
+    isPokemon pokemon;
+
+    public static void main(String[] args) {
+        App app = new App();
+    }
+
+    public App() {
```
Comment (ibereziuk) [inline 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java]: constructor is for construction purpose. Create for example in class App one public method run() and call it from main()

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        pickPokemon();
+        displayOptions();
+        playerTurn();
+    }
+
+    public void startGame() {
+        System.out.println("Who is that pokemon?!");
+    }
+
+    public void addPokemon() {
```
Comment (ibereziuk) [inline 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java]: addPokemons()? plural version fits better

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/ispokemon/isPokemon.java (hunk 2 above)
Comment (ibereziuk) [reply 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/ispokemon/isPokemon.java]: valid but bad usage of interface :(

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        pokemonList.add(new Squirtle());
+    }
+
+    public void pickPokemon() {
+        pokemon = pokemonList.get(RANDOM.nextInt(pokemonList.size()));
+        pokemon.speak();
+    }
+
+    public void displayOptions() {
+        System.out.println("\n");
```
Comment (ibereziuk) [inline 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java]: be consistent.
call println() twice or call print("\n\n");

File: 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java (hunk 4 above)
Comment (ssoegriemsingh) [reply 3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java]: The pural of pokemon is pokemon..

**Pull request merged**